src/Block/FormatterBlockService.php

```diff
@@ -48,9 +48,8 @@ public function configureEditForm(FormMapper $form, BlockInterface $block): void
         $form->add('settings', ImmutableArrayType::class, [
             'keys' => [
                 ['content', FormatterType::class, static fn (FormBuilderInterface $formBuilder) => [
```

Comment:
The third parameter I think it can be simply an array with your changes.

Reply:
Yeah, It works